### ISSUE TITLE
Dark UI theme

### ISSUE THREAD

**Issue by qwertymodo** [fadden/6502bench#17]
This is a pretty low-priority request, but I tend to set up all of my dev tools with dark UI themes because it's easier on my eyes over long periods.  A dark UI option (or even full user theming) would be really nice.

**Comment by fadden** [comment]
For the most part I'm using default colors for dialogs and controls -- there's a couple of places where I override the background in a read-only TextBox so it's not dark grey -- so a system-wide Windows theme should apply to SourceGen.

I haven't looked into themeifying a WinForms application, so I'm not sure how easy or hard that is.

**Comment by qwertymodo** [comment]
In Windows 10, at least, the system "Dark Mode" only applies to the title bar, and everything else in the UI is the standard black-on-white light theme.  You might want to look into switching to WPF, WinForms is pretty limited in comparison, especially once you start getting into some of the other UI features you mention in your TODO list.  WPF is also hardware-accelerated.

**Comment by fadden** [comment]
I tried "dark mode" just now on Win10.  The settings window itself got very dark, but nothing else changed at all.  Kind of disappointing.

I targeted WinForms for portability to Linux and Mac OS X.  It currently crashes on startup in WinForms code, and Wine-bottling doesn't appear to work, so this idea isn't working out so well in practice.  I'm hoping that tweaks to the code and/or ongoing work on Mono will make it work someday.  Mono has stated they have no plans to support WPF.

**Comment by fadden** [comment]
Moved to https://github.com/fadden/6502bench/wiki/TO-DO-List

Using WPF now.  Changing the window colors is easy enough, but I've got a bunch of custom styling for things like long comments that span multiple ListView columns, so it's not trivial.  (I think.  Still not exactly a WPF expert.)

I may need to do something clever with Note highlight colors, since they're currently chosen to contrast with black text.  (Probably just need a luminance inversion.)
